src/input.cpp

```diff
@@ -80,10 +80,10 @@ bool is_mouse_enabled()
 #endif
 }
 
-static bool is_keycode_mode_supported()
+bool is_keycode_mode_supported()
 {
 #if defined(TILES) && !defined(__ANDROID__)
-    return true;
+    return get_option<std::string>( "SDL_KEYBOARD_MODE" ) == "keycode";
```

Comment:
I suggest making it a global var to cache it, like we do in `void options_manager::load()`:

https://github.com/CleverRaven/Cataclysm-DDA/blob/b3a77a0519d4cada44984ef7a9849eb972868bf0/src/options.cpp#L3062